lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1154,13 +1154,8 @@ protected function prepare_links( $post ) {
 					continue;
 				}
 
-				if ( 'post_tag' === $tax ) {
```

Comment:
@jdolan #1381 still stands, that's not about what this fix was for. This is about using rest_base to build the url. If #1381 means we set `rest_base` to be `post_tag` then fine. This code is right either way.